npm-packages/meteor-installer/install.js

```diff
@@ -74,7 +71,13 @@ function download() {
   }, cliProgress.Presets.shades_classic);
   downloadProgress.start(100, 0);
 
-  const url = `https://packages.meteor.com/bootstrap-link?arch=os.windows.x86_64&release=${release}`
+  const downloadPlatform = {
```

Comment:
this is a constant (should be UPPER_CASE) and could be declared in the file scope.